QuipEditor.py

```diff
@@ -90,29 +96,79 @@ def _print_tree(self, node, prefix, postfix):
 		return str_result
 
 
+class ShowContactsCommand(sublime_plugin.WindowCommand):
+	def run(self):
+		view = self.window.new_file()
+		view.run_command(COMMAND_INSERT_CONTACTS)
+
+
+class InsertContactsCommand(sublime_plugin.TextCommand):
+
+	def run(self, edit):
+		self.view.erase_phantoms(KEY_CONTACTS_PHANTOM_SET)
+		self.view.set_read_only(True)
+		self.view.set_name("Contacts")
+
+		user, friends = quip.get_contacts()
+		tree = self._print_user(user)
+		tree += "<ul>"
+		for friend in friends:
+			tree += "<li>%s</li>" % self._print_user(friend)
+		tree += "</ul>"
+
+		phantom = sublime.Phantom(
+			region=Region(0, self.view.size()),
+			content=tree,
+			layout=sublime.LAYOUT_INLINE,
+			on_navigate=self._open_chat
+		)
+		sublime.PhantomSet(self.view, KEY_CONTACTS_PHANTOM_SET).update([phantom, phantom])
+
+	def _print_user(self, user):
```

Comment:
Move it into user class as method toHTML

Reply:
Use pythonic function name  `to_html`
Or place the logic to `__str__` method

Reply:
done

Reply:
In my view html views of User is not represent string form of User, it's more specific form so to_html is more correct here